lib/rubocop/cop/rspec/example_wording.rb

```diff
@@ -53,27 +53,30 @@ def on_block(node)
           end
         end
 
-        def autocorrect(range)
-          ->(corrector) { corrector.replace(range, replacement_text(range)) }
+        def autocorrect(node)
+          lambda do |corrector|
```

Comment:
It’s a bit unfortunate that when switching from a one-line to multi-line block, we have to also change how the `lambda` is defined. That’s not diff friendly…

Reply:
Indeed it would be very nice if git knew language semantics. Just checked all four diff algorithms, none showed anything that would hint that `range` became `node` in the signature, `docstring(node)` in method call. /shrug